lib/styles.js

```diff
@@ -17,6 +18,11 @@ export default function(val, props) {
     clientWidth = document.body.clientWidth;
     if (/\D/.test(width))
       width = document.body.clientWidth * (width.match(/\d*/) / 100);
+    if (open) {
+      document.body.style.overflowY = 'hidden';
+    } else {
+      document.body.style.overflowY = 'auto';
```

Comment:
Not a fan of modifying the overflow styles of `body`. This is a very invasive way to go about this. Can you think of any alternatives to this? I'll try and do some testing this week to this regard as well...